### PR TITLE
Add a prop 'isPinned' to the VideoTile component for toggling the pin icon

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Add a prop 'isPinned' to the VideoTile component for toggling the pin icon

Screenshot 2022-11-09 at 8 43 49 PM

Screenshot 2022-11-09 at 8 44 00 PM

Participant with long name
Screenshot 2022-11-14 at 9 55 15 PM

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3015605

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/VideoTile.tsx [hunk 1]
```diff
@@ -86,6 +86,11 @@ export interface VideoTileProps {
    * Whether the video is muted or not.
    */
   isMuted?: boolean;
+  /* @conditional-compile-remove(pinned-participants) */
+  /**
+   * If true, the video tile will be rendered with a pin icon.
```
Comment (anjulgarg) [inline packages/react-components/src/components/VideoTile.tsx:91]: Slightly misleading. This has nothing to do with the video.
If true, the VideoTile will show a pin icon.

File: packages/react-components/src/components/VideoTile.tsx [hunk 2]
```diff
@@ -295,6 +302,19 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
                   <Icon iconName="VideoTileMicOff" />
                 </Stack>
               )}
+              {
+                /* @conditional-compile-remove(pinned-participants) */
+                isPinned && (
+                  <Stack className={mergeStyles(iconContainerStyle)}>
+                    <Icon
+                      iconName="PinnedSolid"
```
Comment (anjulgarg) [inline packages/react-components/src/components/VideoTile.tsx:310]: We need to make this icon customizable by Contoso

Comment (prabhjot-msft) [reply packages/react-components/src/components/VideoTile.tsx]: done

File: packages/react-components/src/components/VideoTile.tsx [hunk 3]
```diff
@@ -295,6 +302,19 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
                   <Icon iconName="VideoTileMicOff" />
                 </Stack>
               )}
+              {
+                /* @conditional-compile-remove(pinned-participants) */
+                isPinned && (
+                  <Stack className={mergeStyles(iconContainerStyle)}>
+                    <Icon
+                      iconName="VideoTilePinned"
+                      className={mergeStyles({
```
Comment (anjulgarg) [inline packages/react-components/src/components/VideoTile.tsx:311]: Let's move this style to the VideoTile.styles file

**Review by anjulgarg — APPROVED** [review]

File: packages/react-components/src/components/VideoTile.tsx (hunk 3 above)
Comment (prabhjot-msft) [reply packages/react-components/src/components/VideoTile.tsx:311]: done

**Pull request merged**